Layout.py

```diff
@@ -151,7 +189,7 @@ def value_cells_to_layout(self, value_cells):
         layout = {'cols': cols, 'rows': rows, 'cells': cells}
         return layout
 
-    def resize_pane(self, group, direction, step):
+    def resize_pane(self, group, direction, step, alt):
```

Comment:
'alt' stand for alternative: when a pane is surrounded by other panes, in the current version pressing left or right will always expand the pane: it will pull left the left border, and right the right border. With the changes I made, pressing left or right always pulls the same border, if you want to pull the other one you use the keybinding with the 'alt' argument set to true.

This is an example:
http://i.imgur.com/WISmYYy.gif

For each direction, first the normal keybinding is used (Alt+hjkl for me), then the 'alt true' (for me, Ctrl+Alt+hjkl) keybinding is used.